cmd/config-sync/config_sync.go

```diff
@@ -31,7 +32,7 @@ type ConfigSync struct {
 	vanClient *client.VanClient
 }
 
-const SHARED_TLS_DIRECTORY = "/etc/skupper-router-certs"
+const SHARED_TLS_DIRECTORY = "/etc/skupper-config-router-certs"
```

Comment:
What do you think of using something that makes it clear it has certs for links only, like: `skupper-router-link-certs` ?

Reply:
You are right about having a more explicit name, but It is going to be also used for all the certificates that the config-sync sidecar downloads for the router, links and httpConnectors/httpListeners (when the tls feature is enabled).

Reply:
You're right. I forgot about the TLS certs. Yes, it sounds clearer to me.